client/src/components/Loading.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '100vh',
+  },
+  circularProgress: {
+    width: '11vh !important',
+    height: '11vh !important',
+  },
+}));
+
+export default function CircularIndeterminate() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <CircularProgress className={classes.circularProgress} />
```

Comment:
Is there a reason to use !important for the width/height? Its worth mentioning that you can provide a size prop for this:
```suggestion
      <CircularProgress className={classes.circularProgress} size="11vh" />
```